src/reuse/modules/service/odata.ts

```diff
@@ -296,10 +296,12 @@ export class OData {
    * };
    * const res = await service.odata.callFunctionImport(srv, "Cancel", options);
    */
-  async callFunctionImport(srv: any, functionImportName: string, options: any): Promise<any> {
+  async callFunctionImport(srv: any, functionImportName: string, options: any, raw: boolean): Promise<any> {
```

Comment:
adding raw like I did on POST before

Reply:
needed for example to retrieve the sap-messages which is used for status information if post was successful or not